morph.js

```diff
@@ -5,9 +5,16 @@ module.exports = morph
 
 // diff elements and apply the resulting patch to the old node
 // todo (yw): copy over events
-// todo (yw): investigate what else to copy over
-// (obj, obj) -> null
+// todo (yw): investigate what else to copy over // (obj, obj) -> null
```

Comment:
Ok cool!
My preference is to remove them since in my experience they end up becoming outdated.
For me not having them encourages me to name variables etc. better so that the code _is_ the documentation.